packages/lit-html/src/directive-helpers.ts

```diff
@@ -173,6 +183,12 @@ export const insertPart = (
   return part;
 };
 
+declare global {
```

Comment:
Augmenting globals is messy and surprising for users, consider instead casting to:

```ts
interface NodeWithMoveBefore extends Node {
  moveBefore?: <T extends Node>(node: T, child: Node | null) => T;
}
```

Reply:
Good point!